### DISCUSSION TITLE
Low moderation

### DISCUSSION THREAD

**Discussion by EagleMind** [mbrlabs/Lorien#285] [Q&A]
It's understandable for someone to get busy in life and not be able to follow up on a project, but is there a specific reason why no PR's are being approved? I recently found out that someone demonstrated a Multiplayer mode but it was not integrated and that was quite surprising to me.
The reason i'm not volenteering myself is that I'm a Js guy and I don't know godot but one could learn it by contributing if the project is this interesting, So i'm asking if someone can take over the PR reviews at least? I think this project has potential, Although much more if it was developed in another language due to community size and app fluidity.

**Comment by mbrlabs** [comment]
Hey, don't worry. I am still maintaining the project and merge PRs if i have time :)
The multiplayer demo was a POC if i remember correctly and definitely out of scope...there are more important issues/features to implement. 

> So i'm asking if someone can take over the PR reviews at least?

Anyone can review PRs as far as i know.

**Reply by tafode** [reply]
@mbrlabs Would it make your work much easier if others did the code reviews? Or would you still look at the code again to make sure that all the rules have been followed?

**Reply by mbrlabs** [reply]
Depends on how familiar the reviewer is with the codebase and Godot in general, but i'd say yes absolutely. I will still look at the code myself though ;)
